src/bp-activity/classes/class-bp-activity-follow.php

```diff
@@ -144,12 +144,20 @@ public function delete() {
 	 *
 	 * @since BuddyBoss 1.0.0
 	 *
-	 * @param int $user_id The user ID.
+	 * @param int   $user_id    The user ID.
+	 * @param array $query_args Query arguments.
+	 * 
```

Comment:
```suggestion
         * @since BuddyBoss [BBVERSION] added support for query arguments.
         *
	 * @param int   $user_id    The user ID.
	 * @param array $query_args Query arguments.
	 * 
```